test/app/AppSettings-test.js

```
@@ -21,6 +21,17 @@ describe('AppSettings', () => {
     assert.ok(new AppSettings().settingsFolder.includes('.sgcloud'))
   })
 
+  it('should throw if application data are not exist', (done) => {
```

should throw an error if application data does not exist